src/seqan2seqan3.hpp

```diff
@@ -51,24 +63,36 @@ inline seqan3::gapped<alph_t> gapValueImpl(seqan3::gapped<alph_t> const *)
 namespace seqan
 {
 
+// cpp20 span does not define const_iterator
+#if defined(__cpp_lib_span)
+template <typename TContainer>
+    requires (!requires { typename TContainer::const_iterator; })
+struct Value<Iter<TContainer const, StdIteratorAdaptor> >
+{
+    typedef TContainer const TContainer_;
+    typedef typename TContainer::iterator const TIterator_;
+    typedef typename std::iterator_traits<TIterator_>::value_type Type;
+};
+#endif
+
 template <typename t>
 inline constexpr bool is_new_range = false;
 template <typename char_t, typename traits_t>
 inline constexpr bool is_new_range<std::basic_string_view<char_t, traits_t>> = true;
-template <typename t, auto j>
+template <typename t, size_t j>
 inline constexpr bool is_new_range<std::span<t, j>> = true;
 template <typename t>
-inline constexpr bool is_new_range<std::span<t, -1>> = true;
+inline constexpr bool is_new_range<std::span<t, std::dynamic_extent>> = true;
 template <typename ...ts>
-inline constexpr bool is_new_range<ranges::transform_view<ts...>> = true;
+inline constexpr bool is_new_range<std::ranges::transform_view<ts...>> = true;
 template <typename ...ts>
-inline constexpr bool is_new_range<ranges::subrange<ts...>> = true;
+inline constexpr bool is_new_range<std::ranges::subrange<ts...>> = true;
 template <typename ...ts>
 inline constexpr bool is_new_range<seqan3::detail::view_translate_join<ts...>> = true;
 template <typename ...ts>
 inline constexpr bool is_new_range<seqan3::detail::view_translate_single<ts...>> = true;
-template <typename t, bool b1, bool b2>
-inline constexpr bool is_new_range<seqan3::detail::view_take<t, b1, b2>> = true;
+template <typename t>
+inline constexpr bool is_new_range<std::ranges::take_view<t>> = true;
```

Comment:
This means `is_new_range` is specialised for the the standard take_view but not anymore for SeqAn's take_view. Shouldn't we have both?

Reply:
We can have both, but as far as I remember this is important because `seqan3::views::slice` is used somewhere. In seqan3 we no use the std views for slice, so we technically don't need the specialisation for seqan3.